cypress/e2e/awx/resources/credentials.cy.ts

```diff
@@ -354,21 +354,27 @@ describe('Create Credentials of different types', () => {
             (item.required as { field: string; dataCy: string }[]).forEach((credentialType) => {
               switch (item.name) {
                 case 'HashiCorp Vault Secret Lookup':
-                  cy.get(`[data-cy="${credentialType.dataCy}"]`).type(`${credentialType.field}`);
+                  cy.getByDataCy(`[data-cy="${credentialType.dataCy}"]`).type(
```

Comment:
hey @vishsanghishetty I believe we decide to remove this specific test in another PR, but the usage of `getByDataCy` would not require to pass `data-cy`again? Just the name of the locator?

See: https://github.com/ansible/ansible-ui/pull/2601

Reply:
@nixocio Yes, I was the one who wrote the creation of 30 different credentials test. I see that it is skipped, fine with me. since this PR is updated with the new changes, it will skip this test. I might have changed this locator when I made bulk changes in the file. hope that answers your question.